tests/src/ExistingSite/ManageSiteSpecificFooterTest.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\Tests\oe_showcase\ExistingSite;
+
+use Drupal\Tests\oe_showcase\Traits\UserTrait;
+
+/**
+ * Tests that a manage_site_specific_footer user can manage footer links.
+ */
+class ManageSiteSpecificFooterTest extends ShowcaseExistingSiteTestBase {
+
+  use UserTrait;
+
+  /**
+   * Test users with manage_site_specific_footer role can manage footer links.
+   */
+  public function testUsersWithManageSiteSpecificFooterRole(): void {
```

Comment:
To simplify, I think the method name should simply describe the operation itself, no need to mention the user role.
```suggestion
  public function testManageFooterAccess(): void {
```